distributions/aws/examples/storage/fsx-for-lustre/README.md

```diff
@@ -0,0 +1,183 @@
+```
+
+You can confirm that FSx CSI Driver was installed using the following command - 
+```
+kubectl get csidriver -A
+
+NAME              ATTACHREQUIRED   PODINFOONMOUNT   MODES        AGE
+fsx.csi.aws.com   false            false            Persistent   14s
+```
+
+## 3.0 Create the IAM Policy for the CSI Driver
+The CSI driver's service account (created during installation) requires IAM permission to make calls to AWS APIs on your behalf. Here, we will be annotating the Service Account `fsx-csi-controller-sa` with an IAM Role which has the required permissions.
+
+1. Create the policy using the json file provided as follows - 
+```
+aws iam create-policy \
+    --policy-name Amazon_FSx_Lustre_CSI_Driver \
+    --policy-document file://fsx-for-lustre/fsx-csi-driver.json
+```
+
+2. Create an IAM role and attach the IAM policy to it. Annotate the Kubernetes service account with the IAM role ARN and the IAM role with the Kubernetes service account name. You can create the role using eksctl as follows - 
+
+```
+eksctl create iamserviceaccount \
+    --name fsx-csi-controller-sa \
+    --namespace kube-system \
+    --cluster $CLUSTER_NAME \
+    --attach-policy-arn arn:aws:iam::$AWS_ACCOUNT_ID:policy/Amazon_FSx_Lustre_CSI_Driver \
+    --region $CLUSTER_REGION \
+    --approve \
+    --override-existing-serviceaccounts 
+```
+
+3. You can verify by describing the specified service account to check if it has been correctly annotated - 
+```
+kubectl describe -n kube-system serviceaccount fsx-csi-controller-sa
+```
+
+## 4.0 Create an Instance of the FSx Filesystem
+Please refer to the official [AWS FSx CSI Document](https://docs.aws.amazon.com/eks/latest/userguide/fsx-csi.html) for detailed instructions on creating an FSx filesystem. 
+
+## 5.0 Using FSx Storage in Kubeflow
+
+## 5.1 Provisioning Options
+### Option 1 - Static Provisioning
+[Using this sample from official Kubeflow Docs](https://www.kubeflow.org/docs/distributions/aws/customizing-aws/storage/#amazon-fsx-for-lustre) 
+
+1. Use the AWS Console to get the filesystem id of the FSx volume you want to use. You could also use the following command to list all the volumes available in your region. Either way, make sure that `file_system_id` is set. 
+```
+aws fsx describe-file-systems --query "FileSystems[*].FileSystemId" --output text --region $CLUSTER_REGION
+```
+
+```
+export file_system_id=<fsx-id-to-use>
+```
+
+2. Once you have the filesystem id, Use the following command to retriev DNSName, and MountName values.
+```
+export dns_name=$(aws fsx describe-file-systems --file-system-ids $file_system_id --query "FileSystems[0].DNSName" --output text --region $CLUSTER_REGION)
+
+export mount_name=$(aws fsx describe-file-systems --file-system-ids $file_system_id --query "FileSystems[0].LustreConfiguration.MountName" --output text --region $CLUSTER_REGION)
+```
+
+3. Now edit the sample/fsx-pv.yaml to replace <file_system_id>, <dns_name>, and <mount_name> with your values.
```

Comment:
nit: `` around file location and pv file location needs updated path

Reply:
done.